github/Requester.py

```diff
@@ -365,20 +432,13 @@ def __createConnection(self):
         ## set.
         ## http_proxy: http://user:password@proxy_host:proxy_port
```

Comment:
Probably we can remove these few lines of comments

Reply:
Good point. I shouldn't be working at this hour ;)